lib/fluent/supervisor.rb

```diff
@@ -591,7 +587,7 @@ def init(process_type, worker_id)
         $log = Fluent::Log.new(logger, @opts)
         $log.enable_color(false) if @path
         $log.enable_debug if @level <= Fluent::Log::LEVEL_DEBUG
-        $log.info "init #{process_type} logger", path: path, rotate_age: @log_rotate_age, rotate_size: @log_rotate_size
+        $log.info "init #{process_type} logger", path: @path, rotate_age: @log_rotate_age, rotate_size: @log_rotate_size
```

Comment:
Memo: I've confirmed that it always outputs `path=nil` even if `--log=/path/to/log` is set:
```
2023-02-13 18:38:40 +0900 [info]: init supervisor logger path=nil rotate_age=nil rotate_size=nil
```

Reply:
Really? I don't understand it. I will check it.

Reply:
Ah, sorry. I was talking about the case without this fix.

Reply:
I see! When we don't enable log rotate, this always outputs `path=nil`.

Reply:
This PR fixes this problem too.